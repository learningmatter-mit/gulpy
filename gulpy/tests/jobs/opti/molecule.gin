opti conp
title
 
end

cartesian
  C_3  core    4.76350    3.56507    8.39721
  N_3  core    6.14079    3.81175    7.85097
  C_3  core    7.03495    2.78737    8.47416
  C_3  core    6.56797    5.14369    8.39538
  C_3  core    6.19460    3.75862    6.31222
  C_3  core    4.95883    4.47693    5.66443
  C_3  core    5.00532    4.44836    4.12645
  C_3  core    5.00116    2.98210    3.64603
  C_3  core    6.25468    2.27458    4.18649
  C_3  core    7.54686    2.96706    3.70630
  C_3  core    7.50454    4.42956    4.18013
  C_3  core    6.27089    5.17484    3.63813
  C_3  core    7.47448    4.45905    5.72741
  C_3  core    6.20375    2.29436    5.73655
   H_  core    4.08748    4.40541    8.21571
   H_  core    4.34461    2.67235    7.95822
   H_  core    4.77411    3.41351    9.46352
   H_  core    7.13908    2.98292    9.52157
   H_  core    8.03016    2.82281    8.05266
   H_  core    6.64126    1.77285    8.37242
   H_  core    6.27074    5.27555    9.42334
   H_  core    7.65068    5.28499    8.34320
   H_  core    6.09498    5.93806    7.84770
   H_  core    4.03560    3.97981    5.95399
   H_  core    4.90878    5.50988    5.99773
   H_  core    4.14001    4.94938    3.73924
   H_  core    4.09838    2.47223    3.97718
   H_  core    5.03414    2.95404    2.57330
   H_  core    6.26606    1.27754    3.79028
   H_  core    7.60998    2.93990    2.63014
   H_  core    8.42559    2.46037    4.10430
   H_  core    8.36264    4.92855    3.78343
   H_  core    6.30010    5.16146    2.56283
   H_  core    6.26831    6.21335    3.96754
   H_  core    7.49245    5.50365    6.01551
   H_  core    8.37001    3.98586    6.12182
   H_  core    5.32162    1.76339    6.08005
   H_  core    7.07727    1.75451    6.08835

connect 1 2 single
connect 2 3 single
connect 2 4 single
connect 2 5 single
connect 5 6 single
connect 6 7 single
connect 7 8 single
connect 8 9 single
connect 9 10 single
connect 10 11 single
connect 11 12 single
connect 11 13 single
connect 9 14 single
connect 13 5 single
connect 14 5 single
connect 12 7 single
connect 1 15 single
connect 1 16 single
connect 1 17 single
connect 3 18 single
connect 3 19 single
connect 3 20 single
connect 4 21 single
connect 4 22 single
connect 4 23 single
connect 6 24 single
connect 6 25 single
connect 7 26 single
connect 8 27 single
connect 8 28 single
connect 9 29 single
connect 10 30 single
connect 10 31 single
connect 11 32 single
connect 12 33 single
connect 12 34 single
connect 13 35 single
connect 13 36 single
connect 14 37 single
connect 14 38 single

species
H_     core
C_3    core
N_3    core
keyword molmec
element
mass H_       1.00800
mass C_3     12.01100
mass N_3     14.00670
end
rtol 0.0
element
cova H     0.420
cova B     0.800
cova C     0.770
cova N     0.702
cova O     0.800
cova F     0.611
cova Al    1.047
cova Si    0.937
cova P     0.890
cova S     1.040
cova Ga    1.210
cova Ge    1.210
cova As    1.210
cova Se    1.210
cova In    1.390
cova Sn    1.373
cova Sb    1.432
cova Te    1.280
cova I     1.360
end
harmonic bond kcal
H_       H_           700.0    0.6500
C_3      H_           700.0    1.0900
C_3      C_3          700.0    1.5300
N_3      H_           700.0    1.0220
N_3      C_3          700.0    1.4620
N_3      N_3          700.0    1.3940
three-body bond kcal
C_3      X X   100.0  109.4710
N_3      X X   100.0  106.7000
torsion bond kcal dreiding single
X     C_3   C_3   X     1.0    -3 180.00
X     N_3   C_3   X     1.0    -3 180.00
X     N_3   N_3   X     1.0    -3 180.00
epsilon kcal
H_    0.0152 3.1950
C_3   0.0951 3.8983
N_3   0.0774 3.6621
lennard epsilon 12 6 kcal x13 geometric all
0.0 12.5
hydrogen-bond kcal 12 10 4 taper dreiding
H_     N_3   N_3  3741298.1709492207 593660.5362167358 105.0 115.0 &
0.0 3.0 0.0 3.0 0.0 5.0
