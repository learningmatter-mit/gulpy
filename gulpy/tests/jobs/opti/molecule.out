********************************************************************************
*                       GENERAL UTILITY LATTICE PROGRAM                        *
*                                 Julian Gale                                  *
*                       Curtin Institute for Computation                       *
*                           Department of Chemistry                            *
*                    Curtin University, Western Australia                      *
********************************************************************************
* Version = 5.1.1 * Last modified =  11th April 2019                           *
********************************************************************************
*  optimise     - perform optimisation run                                     *
*  conp         - constant pressure calculation                                *
*  molmec       - molecule option activated, Coulomb subtract 1-2/1-3 terms    *
********************************************************************************
*                                                                              *
********************************************************************************


  Job Started  at 09:38.45 11th July       2020                               

  Number of CPUs =     1


  Total number of configurations input =      1

********************************************************************************
*  Input for Configuration =   1                                               *
********************************************************************************

  Formula = C13NH24                                                     

  Number of irreducible atoms/shells =      38


  Total number atoms/shells =      38

  Dimensionality = 0               :  Cluster


  Charge on cluster =   0.000000


  Initial cluster dipoles : x =      0.000000 e.Angs
                            y =      0.000000 e.Angs
                            z =      0.000000 e.Angs

  Temperature of configuration =  0.000     K 

  Symmetry number =      1

  Cartesian coordinates of cluster :

--------------------------------------------------------------------------------
   No.  Atomic       x           y          z         Charge      Occupancy
        Label      (Angs)      (Angs)     (Angs)        (e)         (Frac)  
--------------------------------------------------------------------------------
      1 C3    c      4.7635 *    3.5651 *    8.3972 *   0.00000    1.000000    
      2 N3    c      6.1408 *    3.8117 *    7.8510 *   0.00000    1.000000    
      3 C3    c      7.0349 *    2.7874 *    8.4742 *   0.00000    1.000000    
      4 C3    c      6.5680 *    5.1437 *    8.3954 *   0.00000    1.000000    
      5 C3    c      6.1946      3.7586      6.3122     0.00000    1.000000    
      6 C3    c      4.9588 *    4.4769 *    5.6644 *   0.00000    1.000000    
      7 C3    c      5.0053 *    4.4484 *    4.1264 *   0.00000    1.000000    
      8 C3    c      5.0012 *    2.9821 *    3.6460 *   0.00000    1.000000    
      9 C3    c      6.2547 *    2.2746 *    4.1865 *   0.00000    1.000000    
     10 C3    c      7.5469 *    2.9671 *    3.7063 *   0.00000    1.000000    
     11 C3    c      7.5045 *    4.4296 *    4.1801 *   0.00000    1.000000    
     12 C3    c      6.2709 *    5.1748 *    3.6381 *   0.00000    1.000000    
     13 C3    c      7.4745 *    4.4591 *    5.7274 *   0.00000    1.000000    
     14 C3    c      6.2038 *    2.2944 *    5.7366 *   0.00000    1.000000    
     15 H     c      4.0875 *    4.4054 *    8.2157 *   0.00000    1.000000    
     16 H     c      4.3446 *    2.6723 *    7.9582 *   0.00000    1.000000    
     17 H     c      4.7741 *    3.4135 *    9.4635 *   0.00000    1.000000    
     18 H     c      7.1391 *    2.9829 *    9.5216 *   0.00000    1.000000    
     19 H     c      8.0302 *    2.8228 *    8.0527 *   0.00000    1.000000    
     20 H     c      6.6413 *    1.7729 *    8.3724 *   0.00000    1.000000    
     21 H     c      6.2707 *    5.2755 *    9.4233 *   0.00000    1.000000    
     22 H     c      7.6507 *    5.2850 *    8.3432 *   0.00000    1.000000    
     23 H     c      6.0950 *    5.9381 *    7.8477 *   0.00000    1.000000    
     24 H     c      4.0356 *    3.9798 *    5.9540 *   0.00000    1.000000    
     25 H     c      4.9088 *    5.5099 *    5.9977 *   0.00000    1.000000    
     26 H     c      4.1400 *    4.9494 *    3.7392 *   0.00000    1.000000    
     27 H     c      4.0984 *    2.4722 *    3.9772 *   0.00000    1.000000    
     28 H     c      5.0341 *    2.9540 *    2.5733 *   0.00000    1.000000    
     29 H     c      6.2661 *    1.2775 *    3.7903 *   0.00000    1.000000    
     30 H     c      7.6100 *    2.9399 *    2.6301 *   0.00000    1.000000    
     31 H     c      8.4256 *    2.4604 *    4.1043 *   0.00000    1.000000    
     32 H     c      8.3626 *    4.9285 *    3.7834 *   0.00000    1.000000    
     33 H     c      6.3001 *    5.1615 *    2.5628 *   0.00000    1.000000    
     34 H     c      6.2683 *    6.2134 *    3.9675 *   0.00000    1.000000    
     35 H     c      7.4924 *    5.5037 *    6.0155 *   0.00000    1.000000    
     36 H     c      8.3700 *    3.9859 *    6.1218 *   0.00000    1.000000    
     37 H     c      5.3216 *    1.7634 *    6.0800 *   0.00000    1.000000    
     38 H     c      7.0773 *    1.7545 *    6.0884 *   0.00000    1.000000    
--------------------------------------------------------------------------------



  Molecule list generated from bond lengths :

  Total number of molecules =     1

--------------------------------------------------------------------------------
Molecule No./:  Atoms
Periodicity  :  
--------------------------------------------------------------------------------
     1  0    : C3   c     1 N3   c     2 C3   c     3 C3   c     4 C3   c     5
             : C3   c     6 C3   c     7 C3   c     8 C3   c     9 C3   c    10
             : C3   c    11 C3   c    12 C3   c    13 C3   c    14 H    c    15
             : H    c    16 H    c    17 H    c    18 H    c    19 H    c    20
             : H    c    21 H    c    22 H    c    23 H    c    24 H    c    25
             : H    c    26 H    c    27 H    c    28 H    c    29 H    c    30
             : H    c    31 H    c    32 H    c    33 H    c    34 H    c    35
             : H    c    36 H    c    37 H    c    38
--------------------------------------------------------------------------------


********************************************************************************
*  General input information                                                   *
********************************************************************************

  Species output for all configurations : 

--------------------------------------------------------------------------------
  Species    Type    Atomic    Atomic    Charge       Radii (Angs)     Library
                     Number     Mass       (e)     Cova   Ionic  VDW   Symbol
--------------------------------------------------------------------------------
    H        Core        1       1.01   0.000000   0.420  0.000  1.080          
    C3       Core        6      12.01   0.000000   0.770  0.000  1.530          
    N3       Core        7      14.01   0.000000   0.702  0.000  1.480          
--------------------------------------------------------------------------------


  Accuracy factor for short range sums = 12.000

  Time limit = Infinity

  Maximum range for interatomic potentials =    100000.000000 Angstroms

  General interatomic potentials :

--------------------------------------------------------------------------------
Atom  Types   Potential      Parameter       Value         Units   Cutoffs(Ang)
  1     2                                                            Min /  Max 
--------------------------------------------------------------------------------
H    c H    c Lenn/es       Lennard epsilon 0.65913937E-03eV        3Bond 12.500
                            Lennard sigma/m  3.1950000    Ang       
                            Exponent m        12          None
                            Exponent n         6          None
--------------------------------------------------------------------------------
H    c C3   c Lenn/es       Lennard epsilon 0.16487155E-02eV        3Bond 12.500
                            Lennard sigma/m  3.5466500    Ang       
                            Exponent m        12          None
                            Exponent n         6          None
--------------------------------------------------------------------------------
C3   c C3   c Lenn/es       Lennard epsilon 0.41239575E-02eV        3Bond 12.500
                            Lennard sigma/m  3.8983000    Ang       
                            Exponent m        12          None
                            Exponent n         6          None
--------------------------------------------------------------------------------
H    c N3   c Lenn/es       Lennard epsilon 0.14873937E-02eV        3Bond 12.500
                            Lennard sigma/m  3.4285500    Ang       
                            Exponent m        12          None
                            Exponent n         6          None
--------------------------------------------------------------------------------
C3   c N3   c Lenn/es       Lennard epsilon 0.37204408E-02eV        3Bond 12.500
                            Lennard sigma/m  3.7802000    Ang       
                            Exponent m        12          None
                            Exponent n         6          None
--------------------------------------------------------------------------------
N3   c N3   c Lenn/es       Lennard epsilon 0.33564070E-02eV        3Bond 12.500
                            Lennard sigma/m  3.6621000    Ang       
                            Exponent m        12          None
                            Exponent n         6          None
--------------------------------------------------------------------------------

  Intramolecular potentials :

--------------------------------------------------------------------------------
Atom  Types   Potential      Parameter       Value         Units   Cutoffs(Ang)
  1     2                                                            Min /  Max 
--------------------------------------------------------------------------------
H    c H    c Harmonic      Harmonic k2      30.355102    eV*Ang^-2 0.000 1 Bond
                            Harmonic r0     0.65000000    Ang       
                            Harmonic k3      0.0000000    eV*Ang^-3 
                            Harmonic k4      0.0000000    eV*Ang^-4 
--------------------------------------------------------------------------------
H    c C3   c Harmonic      Harmonic k2      30.355102    eV*Ang^-2 0.000 1 Bond
                            Harmonic r0      1.0900000    Ang       
                            Harmonic k3      0.0000000    eV*Ang^-3 
                            Harmonic k4      0.0000000    eV*Ang^-4 
--------------------------------------------------------------------------------
C3   c C3   c Harmonic      Harmonic k2      30.355102    eV*Ang^-2 0.000 1 Bond
                            Harmonic r0      1.5300000    Ang       
                            Harmonic k3      0.0000000    eV*Ang^-3 
                            Harmonic k4      0.0000000    eV*Ang^-4 
--------------------------------------------------------------------------------
H    c N3   c Harmonic      Harmonic k2      30.355102    eV*Ang^-2 0.000 1 Bond
                            Harmonic r0      1.0220000    Ang       
                            Harmonic k3      0.0000000    eV*Ang^-3 
                            Harmonic k4      0.0000000    eV*Ang^-4 
--------------------------------------------------------------------------------
C3   c N3   c Harmonic      Harmonic k2      30.355102    eV*Ang^-2 0.000 1 Bond
                            Harmonic r0      1.4620000    Ang       
                            Harmonic k3      0.0000000    eV*Ang^-3 
                            Harmonic k4      0.0000000    eV*Ang^-4 
--------------------------------------------------------------------------------
N3   c N3   c Harmonic      Harmonic k2      30.355102    eV*Ang^-2 0.000 1 Bond
                            Harmonic r0      1.3940000    Ang       
                            Harmonic k3      0.0000000    eV*Ang^-3 
                            Harmonic k4      0.0000000    eV*Ang^-4 
--------------------------------------------------------------------------------

  General Three-body potentials :

--------------------------------------------------------------------------------
  Atoms       Potential      Parameter       Value         Units   Cutoffs(Ang)
  1 / 2 / 3                                                          Min /  Max 
--------------------------------------------------------------------------------
H     core    Hydrogen bond Three-body A     162239.27    eV*Ang^m  0.000  3.000
N3    core                  Three-body B     25743.752    eV*Ang^n  0.000  3.000
N3    core                                                          0.000  5.000
--------------------------------------------------------------------------------

  Intramolecular Three-body potentials :

--------------------------------------------------------------------------------
  Atoms       Potential      Parameter       Value         Units   Cutoffs(Ang)
  1 / 2 / 3                                                          Min /  Max 
--------------------------------------------------------------------------------
C3    core    Harmonic      Three-body cnst  4.3364432    eV*rad^-2 0.000 1 Bond
X     core                  Three-body angl  109.47100    degrees   0.000 1 Bond
X     core                  Three-body k4    0.0000000    eV*rad^-4 0.000 1 Bond
                            Three-body k3    0.0000000    eV*rad^-3 
--------------------------------------------------------------------------------
N3    core    Harmonic      Three-body cnst  4.3364432    eV*rad^-2 0.000 1 Bond
X     core                  Three-body angl  106.70000    degrees   0.000 1 Bond
X     core                  Three-body k4    0.0000000    eV*rad^-4 0.000 1 Bond
                            Three-body k3    0.0000000    eV*rad^-3 
--------------------------------------------------------------------------------

  Intramolecular Four-body potentials :

--------------------------------------------------------------------------------
  Atoms       Potential      Parameter       Value         Units   Cutoffs(Ang)
  1 - 4                                                             Min /  Max 
--------------------------------------------------------------------------------
X     core    Standard      Torsional const 0.43364432E-01eV         Bonded
C3    core Phase = -  3                                              single   
C3    core                                                              
X     core
--------------------------------------------------------------------------------
X     core    Standard      Torsional const 0.43364432E-01eV         Bonded
N3    core Phase = -  3                                              single   
C3    core                                                              
X     core
--------------------------------------------------------------------------------
X     core    Standard      Torsional const 0.43364432E-01eV         Bonded
N3    core Phase = -  3                                              single   
N3    core                                                              
X     core
--------------------------------------------------------------------------------

********************************************************************************
*  Output for configuration   1                                                *
********************************************************************************


  Components of energy : 

--------------------------------------------------------------------------------
  Interatomic potentials     =           3.00710091 eV    
  Three-body potentials      =           0.25037054 eV    
  Four-body potentials       =           0.05099294 eV    
  Improper torsions          =           0.00000000 eV    
  Out of plane potentials    =           0.00000000 eV    
  Monopole - monopole (real) =           0.00000000 eV    
--------------------------------------------------------------------------------
  Total lattice energy       =           3.30846439 eV    
--------------------------------------------------------------------------------
  Total lattice energy       =             319.2162 kJ/mol
--------------------------------------------------------------------------------


  Number of variables =      111

  Maximum number of calculations  =          1000
  Maximum Hessian update interval =            10
  Maximum step size               =   1.000000000
  Maximum parameter tolerance     =   0.000010000
  Maximum function  tolerance     =   0.000010000
  Maximum gradient  tolerance     =   0.001000000
  Maximum gradient  component     =   0.010000000

  Symmetry not applied to optimisation

  Newton-Raphson optimiser to be used

  BFGS hessian update to be used

  Lower half triangular hessian to be used

  Start of cluster optimisation :

  Cycle:      0 Energy:         3.308464  Gnorm:      0.023370  CPU:    0.012
  ** Hessian calculated **
  Cycle:      1 Energy:         2.944797  Gnorm:      0.008834  CPU:    0.016
  Cycle:      2 Energy:         2.919185  Gnorm:      0.003656  CPU:    0.018
  Cycle:      3 Energy:         2.916386  Gnorm:      0.001071  CPU:    0.020
  Cycle:      4 Energy:         2.913670  Gnorm:      0.001996  CPU:    0.022
  Cycle:      5 Energy:         2.912385  Gnorm:      0.001691  CPU:    0.024
  Cycle:      6 Energy:         2.911581  Gnorm:      0.000781  CPU:    0.026
  Cycle:      7 Energy:         2.911335  Gnorm:      0.000425  CPU:    0.027
  Cycle:      8 Energy:         2.911268  Gnorm:      0.000283  CPU:    0.028
  Cycle:      9 Energy:         2.911241  Gnorm:      0.000186  CPU:    0.030


  **** Optimisation achieved ****


  Final energy =       2.91123336 eV
  Final Gnorm  =       0.00005011

  Components of energy : 

--------------------------------------------------------------------------------
  Interatomic potentials     =           2.58017698 eV    
  Three-body potentials      =           0.31323281 eV    
  Four-body potentials       =           0.01782357 eV    
  Improper torsions          =           0.00000000 eV    
  Out of plane potentials    =           0.00000000 eV    
  Monopole - monopole (real) =           0.00000000 eV    
--------------------------------------------------------------------------------
  Total lattice energy       =           2.91123336 eV    
--------------------------------------------------------------------------------
  Total lattice energy       =             280.8895 kJ/mol
--------------------------------------------------------------------------------

  Final cartesian coordinates of atoms :

--------------------------------------------------------------------------------
   No.  Atomic        x           y          z          Radius
        Label       (Angs)      (Angs)     (Angs)       (Angs) 
--------------------------------------------------------------------------------
     1  C3    c     4.830110    3.324808    8.455088    0.000000
     2  N3    c     6.161905    3.788994    7.881560    0.000000
     3  C3    c     7.220745    2.895686    8.513159    0.000000
     4  C3    c     6.397724    5.180814    8.451904    0.000000
     5  C3    c     6.194600    3.758620    6.312220    0.000000
     6  C3    c     4.945194    4.483277    5.647685    0.000000
     7  C3    c     4.979264    4.459574    4.085763    0.000000
     8  C3    c     4.969565    2.981314    3.591792    0.000000
     9  C3    c     6.226391    2.252318    4.154536    0.000000
    10  C3    c     7.518910    2.958961    3.645259    0.000000
    11  C3    c     7.514742    4.436779    4.139009    0.000000
    12  C3    c     6.264519    5.178019    3.576141    0.000000
    13  C3    c     7.477267    4.471708    5.700619    0.000000
    14  C3    c     6.200373    2.284586    5.716433    0.000000
    15  H     c     4.017955    4.016539    8.222224    0.000000
    16  H     c     4.542973    2.340072    8.086151    0.000000
    17  H     c     4.851905    3.250254    9.547530    0.000000
    18  H     c     7.258823    3.003216    9.602398    0.000000
    19  H     c     8.222497    3.121634    8.147489    0.000000
    20  H     c     7.030236    1.836671    8.327245    0.000000
    21  H     c     6.271272    5.210916    9.539288    0.000000
    22  H     c     7.412234    5.537266    8.262431    0.000000
    23  H     c     5.705685    5.916762    8.042423    0.000000
    24  H     c     4.012600    4.003621    5.951342    0.000000
    25  H     c     4.884963    5.524763    5.967577    0.000000
    26  H     c     4.098047    4.976359    3.693026    0.000000
    27  H     c     4.059035    2.476016    3.926954    0.000000
    28  H     c     4.973546    2.949387    2.498315    0.000000
    29  H     c     6.224387    1.213327    3.810382    0.000000
    30  H     c     7.560203    2.926874    2.552551    0.000000
    31  H     c     8.406514    2.439420    4.018054    0.000000
    32  H     c     8.420772    4.937720    3.783894    0.000000
    33  H     c     6.287484    5.178398    2.482436    0.000000
    34  H     c     6.265999    6.222865    3.900202    0.000000
    35  H     c     7.513433    5.522539    5.994632    0.000000
    36  H     c     8.393000    4.007367    6.070015    0.000000
    37  H     c     5.323668    1.727296    6.050269    0.000000
    38  H     c     7.077224    1.731698    6.059157    0.000000
--------------------------------------------------------------------------------


  Final Cartesian derivatives :

--------------------------------------------------------------------------------
   No.  Atomic          x             y             z           Radius
        Label       (eV/Angs)     (eV/Angs)    (eV/Angs)      (eV/Angs)
--------------------------------------------------------------------------------
      1 C3    c      -0.000292     -0.000816     -0.000127      0.000000
      2 N3    c      -0.000291      0.000225     -0.000349      0.000000
      3 C3    c       0.000673      0.000628      0.000538      0.000000
      4 C3    c       0.001260      0.000221      0.001868      0.000000
      5 C3    c       0.000000      0.000000      0.000000      0.000000
      6 C3    c      -0.000169     -0.000099     -0.000341      0.000000
      7 C3    c      -0.000016      0.000038     -0.000083      0.000000
      8 C3    c       0.000083      0.000009     -0.000147      0.000000
      9 C3    c       0.000130      0.001458     -0.000096      0.000000
     10 C3    c      -0.000078      0.000002     -0.000096      0.000000
     11 C3    c      -0.001363     -0.000068      0.000149      0.000000
     12 C3    c      -0.000151     -0.000013     -0.000170      0.000000
     13 C3    c       0.000058     -0.000332      0.000010      0.000000
     14 C3    c      -0.000039      0.000198     -0.000434      0.000000
     15 H     c      -0.000182      0.000627     -0.000116      0.000000
     16 H     c      -0.001143     -0.000644     -0.000636      0.000000
     17 H     c       0.000693      0.000096      0.001044      0.000000
     18 H     c      -0.000315      0.000586      0.001007      0.000000
     19 H     c       0.001345     -0.000708     -0.000678      0.000000
     20 H     c      -0.000798     -0.001180     -0.000095      0.000000
     21 H     c       0.000252      0.000439     -0.001153      0.000000
     22 H     c      -0.000874      0.001046     -0.000392      0.000000
     23 H     c      -0.000473     -0.000191      0.001084      0.000000
     24 H     c       0.000186      0.000011     -0.000201      0.000000
     25 H     c      -0.000106     -0.000101     -0.000215      0.000000
     26 H     c       0.000368     -0.000196      0.000035      0.000000
     27 H     c       0.000085     -0.000004     -0.000037      0.000000
     28 H     c       0.000091     -0.000242      0.000201      0.000000
     29 H     c      -0.000169     -0.001045      0.000271      0.000000
     30 H     c      -0.000343      0.000086     -0.000074      0.000000
     31 H     c       0.000157      0.000004      0.000058      0.000000
     32 H     c       0.000930      0.000185      0.000311      0.000000
     33 H     c       0.000246     -0.000078      0.000329      0.000000
     34 H     c       0.000008     -0.000136     -0.000001      0.000000
     35 H     c       0.000223     -0.000103     -0.000414      0.000000
     36 H     c      -0.000005      0.000208     -0.000106      0.000000
     37 H     c       0.000202     -0.000078     -0.000310      0.000000
     38 H     c      -0.000136     -0.000028     -0.000384      0.000000
--------------------------------------------------------------------------------
  Maximum abs         0.001363      0.001458      0.001868      0.000000
--------------------------------------------------------------------------------


  Time to end of optimisation =       0.0324 seconds


  Peak dynamic memory used =       0.90 MB 


  Timing analysis for GULP :

--------------------------------------------------------------------------------
  Task / Subroutine                                          Time (Seconds)
--------------------------------------------------------------------------------
  Calculation of real space energy and derivatives                0.0138
  Calculation of three-body energy and derivatives                0.0021
  Calculation of four-body energy and derivatives                 0.0038
  Calculation of molecules and connectivity                       0.0001
  Calculation and manipulation of Hessian matrix                  0.0002
  Calculation of matrix inversion                                 0.0014
--------------------------------------------------------------------------------
  Total CPU time                                                  0.0324
--------------------------------------------------------------------------------


  Job Finished at 09:38.45 11th July       2020                               

